owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/methods/HttpBaseMethod.kt

```diff
@@ -13,24 +13,39 @@ import java.net.MalformedURLException
 import java.net.URL
 import java.util.concurrent.TimeUnit
 
-abstract class HttpBaseMethod constructor(clientWrapper: HttpClient, url: URL) {
-    var okHttpClient: OkHttpClient
+abstract class HttpBaseMethod constructor(url: URL) {
     var httpUrl: HttpUrl = url.toHttpUrlOrNull() ?: throw MalformedURLException()
     var request: Request
     abstract var response: Response
-
     var call: Call? = null
 
+    var followRedirects: Boolean? = true
+    var retryOnConnectionFailure: Boolean? = false
```

Comment:
If we use default values, could we remove the nullability?

```suggestion
    var followRedirects: Boolean = true
    var retryOnConnectionFailure: Boolean = false
```